sdk/communication/communication-administration/review/communication-administration.api.md

```diff
@@ -270,26 +278,24 @@ export interface PageableOptions extends OperationOptions {
 export class PhoneNumberAdministrationClient {
     constructor(connectionString: string, options?: PhoneNumberAdministrationClientOptions);
     constructor(url: string, credential: KeyCredential, options?: PhoneNumberAdministrationClientOptions);
-    cancelSearch(searchId: string, options?: CancelSearchOptions): Promise<VoidResponse>;
+    beginCancelSearch(searchId: string, options?: BeginCancelSearchOptions): Promise<PollerLike<PollOperationState<PhoneNumberSearch>, PhoneNumberSearch>>;
+    beginCreateSearch(searchRequest: CreateSearchRequest, options?: CreateSearchOptions): Promise<PollerLike<PollOperationState<PhoneNumberSearch>, PhoneNumberSearch>>;
```

Comment:
I don't think this will work because `cancel` for the poller means using an abort signal to kill the request. Cancelling the search is an actual LRO. Here is the definition of `cancel` for the poller: https://github.com/Azure/azure-sdk-for-js/blob/49f21adf3fa4b3359b9d759c85a29f63d9b6583f/sdk/core/core-lro/src/pollOperation.ts#L77